Link to npm package for zen-observable

zen-observable dependency is causing issues during install with docker node image because needs git to download.

Please change the git form
```
"zen-observable": "git+https://github.com/evanshauser/zen-observable.git"
```
to npm form
```
"zen-observable": "^0.5.2"
```

Aw! I'm lost 2 hours just because of this, I'll paste log here for google crawling.
```shell
npm http request GET https://registry.npmjs.org/apollo-fetch
npm info attempt registry request try #1 at 09:11:36
npm http request GET https://registry.npmjs.org/@types%2fzen-observable
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror https://github.com/evanshauser/zen-observable.git /root/.npm/_git-remotes/git-https-github-com-evanshauser-zen-observable-git-5d016840: undefined
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror https://github.com/evanshauser/zen-observable.git /root/.npm/_git-remotes/git-https-github-com-evanshauser-zen-observable-git-5d016840: undefined
npm http 200 https://registry.npmjs.org/apollo-fetch
npm info retry fetch attempt 1 at 09:11:37
npm info attempt registry request try #1 at 09:11:37
npm http fetch GET https://registry.npmjs.org/apollo-fetch/-/apollo-fetch-0.3.0.tgz
npm http fetch 200 https://registry.npmjs.org/apollo-fetch/-/apollo-fetch-0.3.0.tgz
npm http 200 https://registry.npmjs.org/@types%2fzen-observable
npm info addNameTag [ '@types/zen-observable', 'latest' ]
npm info retry fetch attempt 1 at 09:11:38
npm info attempt registry request try #1 at 09:11:38
npm http fetch GET https://registry.npmjs.org/@types/zen-observable/-/zen-observable-0.5.2.tgz
npm http fetch 200 https://registry.npmjs.org/@types/zen-observable/-/zen-observable-0.5.2.tgz
npm ERR! code ENOGIT
npm ERR! not found: git
```
Pretty please fix this, I just don't want to add workaround.

Thanks

--- a/packages/apollo-link-core/package.json
+++ b/packages/apollo-link-core/package.json
@@ -41,7 +41,7 @@
   "dependencies": {
     "graphql": "^0.10.3",
     "graphql-tag": "^2.4.2",
-    "zen-observable": "git+https://github.com/evanshauser/zen-observable.git"
+    "zen-observable-exported": "^0.5.2"
   },
   "devDependencies": {
     "@types/chai": "^4.0.0",

--- a/packages/apollo-link-core/src/index.ts
+++ b/packages/apollo-link-core/src/index.ts
@@ -2,7 +2,7 @@ import { execute, ApolloLink } from './link';
 export { makePromise } from './linkUtils';
 export * from './types';
 
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 
 export default ApolloLink;
 export { Observable, ApolloLink, execute };

--- a/packages/apollo-link-core/src/link.ts
+++ b/packages/apollo-link-core/src/link.ts
@@ -16,7 +16,7 @@ import {
 
 import gql from 'graphql-tag';
 
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 import {
   DocumentNode,
   DefinitionNode,

--- a/packages/apollo-link-core/src/linkUtils.ts
+++ b/packages/apollo-link-core/src/linkUtils.ts
@@ -2,7 +2,7 @@ import { GraphQLRequest, RequestHandler } from './types';
 
 import { ApolloLink, FunctionLink } from './link';
 
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 
 export function validateLink(link: ApolloLink): ApolloLink {
   if (link instanceof ApolloLink && typeof link.request === 'function') {

--- a/packages/apollo-link-core/src/test-utils/mockLink.ts
+++ b/packages/apollo-link-core/src/test-utils/mockLink.ts
@@ -1,6 +1,6 @@
 import { Operation, RequestHandler, NextLink, FetchResult } from '../types';
 
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 
 import { ApolloLink } from '../link';
 

--- a/packages/apollo-link-core/src/test-utils/setContext.ts
+++ b/packages/apollo-link-core/src/test-utils/setContext.ts
@@ -1,6 +1,6 @@
 import { Operation, NextLink, FetchResult } from '../types';
 
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 
 import { ApolloLink } from '../link';
 

--- a/packages/apollo-link-core/src/types.ts
+++ b/packages/apollo-link-core/src/types.ts
@@ -1,4 +1,4 @@
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 import { ExecutionResult, DocumentNode } from 'graphql';
 
 export interface GraphQLRequest {

--- a/packages/apollo-link-core/tests/link.ts
+++ b/packages/apollo-link-core/tests/link.ts
@@ -1,7 +1,7 @@
 import { assert, expect } from 'chai';
 import * as sinon from 'sinon';
 import { execute, ApolloLink } from '../src/link';
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 import { MockLink, SetContextLink } from '../src/test-utils';
 import gql from 'graphql-tag';
 import { FetchResult, Operation } from '../src/types';

--- a/packages/apollo-link-core/tests/linkUtils.ts
+++ b/packages/apollo-link-core/tests/linkUtils.ts
@@ -1,7 +1,7 @@
 import { assert, expect } from 'chai';
 import * as sinon from 'sinon';
 import * as LinkUtils from '../src/linkUtils';
-import * as Observable from 'zen-observable';
+import * as Observable from 'zen-observable-exported';
 
 describe('Link utilities:', () => {
   describe('validateOperation', () => {

